libcloud/compute/base.py

```diff
@@ -545,23 +545,38 @@ class VolumeSnapshot(object):
     """
     A base VolumeSnapshot class to derive from.
     """
-    def __init__(self, id, driver, size=None, extra=None):
+    id = None
+    driver = None
+    size = None
+    extra = None
+    created = None
```

Comment:
I did not know this. My %(deity)s, how often I've used this incorrectly.
Thank you.